src/compose/ComposeBox.js

```diff
@@ -217,6 +220,54 @@ class ComposeBox extends PureComponent<Props, State> {
     }
   };
 
+  insertAttachment = async (attachments: DocumentPickerResponse[]) => {
+    this.setState(({ numUploading }) => ({
+      numUploading: numUploading + 1,
+    }));
+
+    const { _, auth } = this.props;
+
+    const placeholders: string[] = [];
+    const fileNames: string[] = [];
+
+    for (let i = 0; i < attachments.length; i++) {
```

Comment:
tiny nits:
* no blank line here -- that way these are visually joined to the loop that builds them
* swap the order of these declarations, to match the order they get added to in in the loop